RecoLocalCalo/HcalRecAlgos/src/MahiFit.cc

```diff
@@ -78,10 +78,12 @@ void MahiFit::phase1Apply(const HBHEChannelInfo& channelData,
     if (iTS == nnlsWork_.tsOffset)
       tstrig += amplitude;
   }
-
+  const double noisecorr = channelData.noisecorr();
   tsTOT *= channelData.tsGain(0);
   tstrig *= channelData.tsGain(0);
 
+  std::cout << noisecorr << std::endl;
```

Comment:
unprotected cout is verboten in production code.
Please remove.

Reply:
I'm sorry, I forgot to remove the line before PR. The line was removed.